### DISCUSSION TITLE
If put the required property in the FormControl, the FormLabel to * show it.

### DISCUSSION THREAD

**Discussion by wes5510** [channel-io/bezier-react#1802] [Ideas]
### Description

FormControl에 속성중 required=true 를 설정하면 * 표시가 됐으면 좋겠습니다.

### Reasons for suggestion

required 속성만 있고 UI 변경사항이 없는게 어색하다고 생각됩니다.

### Proposed solution

`<FormControl required />` 가 되면 FormLabel 옆에 * 표시가 되고 color도 orange나 red 처럼 붉은색 계열로 되면 좋을거 같습니다!

**Comment by sungik-choi** [comment]
저도 어느정도 동의합니다. 이 스펙이 '서포트봇 폼' 컴포넌트의 라벨에는 구현되어있는데, Bezier의 라벨에는 구현이 안되어있어요(안되어있는데는 모종의 이유가 있지 않을까..? 생각합니다) 

Bezier방에 이슈 레이즈해보시는 것도 좋을 거 같고, 아니면 제가 직접 피드백 전달드려보겠습니다!
